extensions/Worker.Extensions.CosmosDB/release_notes.md

```diff
@@ -4,8 +4,27 @@
 - My change description (#PR/#issue)
 -->
 
-### Microsoft.Azure.Functions.Worker.Extensions.CosmosDB 4.8.1
+### Microsoft.Azure.Functions.Worker.Extensions.CosmosDB 5.0.0
 
-- Updating `Microsoft.Azure.WebJobs.Extensions.CosmosDB` reference to 4.6.1
-- Updating `Microsoft.Extensions.Azure` reference to 1.7.3
-- Updating `Microsoft.Azure.Cosmos` reference to 3.39.1
+- Expose `CosmosClientOptions` to allow configuration of the CosmosDB client (#2483)
+
+#### Breaking Change
+
+- **Default `ConnectionMode` Change:** The default `ConnectionMode` for `CosmosClientOptions` has been changed from `Gateway` to `Direct`.
+  This change is due to `Direct` being the default value for `ConnectionMode` in the Cosmos SDK; now that we are exposing those options,
+  it is not possible for us to tell whether the user has explicitly set the connection mode to `Direct` or not, so we must default to the
+  SDK's default value
+
+##### How to Maintain Previous Behavior
+
+To continue using `Gateway` mode, configure the `CosmosClientOptions` in your `Program` class as shown below:
+
+```csharp
+.ConfigureFunctionsWorkerDefaults((builder) =>
+{
+    builder.Services.Configure<CosmosClientOptions>((options) =>
```

Comment:
I want customers to be able to configure `CosmosClientOptions` directly yes, but I also want it to be clear this configuration is specific for the functions worker cosmos extension. Configuring the options directly is a bit ambiguous what it is for. And what if some other unrelated library also configures cosmos that way?

We should come up with some design where the configuration is exceedingly clear that this is for only the functions worker cosmos extension

Reply:
That is a very valid point, let me think on this a little more.